### ISSUE TITLE
Document trial's interaction with the Python warning system

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3520]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3520|
|Type|enhancement|
|Created|2008-11-01 17:14:05Z|

Basically, document the stuff that happened in [#3487](https://github.com/twisted/twisted/issues/3487).

Attachments:

* [trial-warnings.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/7d8/7d8a209751f64b7e616de77b0e519f77cdd0fc81/2b835b662208a28755f2da1a7bddfcb6db6456ac.patch) (2272 bytes) - added by exarkun on 2008-11-01 17:14:32Z - first pass at some docs

<details><summary>Searchable metadata</summary>

```
trac-id__3520 3520
type__enhancement enhancement
reporter__exarkun exarkun
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__documentation documentation
time__1225559645000000 1225559645000000
changetime__1225641396000000 1225641396000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun removed owner|
|-|-|

Perhaps this is coherent.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph set owner to @exarkun|
|-|-|

Only change I'd make off the top of my head is a hyperlink to "http://docs.python.org/using/cmdline.html#cmdoption-W".  Do that and merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [25280]) Apply trial-warnings.patch with changes based on review

Author: exarkun
Reviewer: glyph
Fixes: [#3520](https://github.com/twisted/twisted/issues/3520)

Add documentation about the interaction between trial and the `warnings` module.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
